Source/WebCore/dom/ContentVisibilityDocumentState.cpp

```diff
@@ -96,50 +94,82 @@ IntersectionObserver* ContentVisibilityDocumentState::intersectionObserver(Docum
     return m_observer.get();
 }
 
-DidUpdateAnyContentRelevancy ContentVisibilityDocumentState::updateRelevancyOfContentVisibilityElements(const OptionSet<ContentRelevancy>& relevancyToCheck)
+bool ContentVisibilityDocumentState::checkRelevancyOfContentVisibilityElement(Element& target, const OptionSet<ContentRelevancy>& relevancyToCheck) const
+{
+    auto oldRelevancy = target.contentRelevancy();
+    OptionSet<ContentRelevancy> newRelevancy;
+    if (oldRelevancy)
+        newRelevancy = *oldRelevancy;
+    auto setRelevancyValue = [&](ContentRelevancy reason, bool value) {
+        if (value)
+            newRelevancy.add(reason);
+        else
+            newRelevancy.remove(reason);
+    };
+    if (relevancyToCheck.contains(ContentRelevancy::OnScreen)) {
+        auto viewportProximity = m_viewportProximity.find(target);
```

Comment:
the return value of `find` is an iterator, not viewportProximity itself.

Reply:
Yeah, that is unclear indeed, fixed.